### PR TITLE
Update troubleshooting bugs.md

### PR DESCRIPTION
**Pull request by flamber**
Pinging @mbvictoria for awesome text inputs :-)

### CODE REVIEW

File: docs/troubleshooting-guide/bugs.md [hunk 1]
```diff
@@ -1,47 +1,33 @@
...
 
 ## Helpful tidbits
 
 ### Accessing the Metabase server logs
-While you can always look for the logs Metabase leaves on your server file system (or however you collect logs), if you are logged into Metabase with an admin account, you can also access them from the drop down menu in the upper right hand corner.
+While you can always look for the logs Metabase leaves on your server file system (or however you collect logs), if you are logged into Metabase with an admin account, you can also access them from the gear drop down menu in the upper right hand corner -> Admin -> Troubleshooting -> Logs.
 
-![Logs Screenshot](images/ServerLogs.png)
+### Checking for Javascript console errors
+Metabase will show debugging information and errors to your browser's developer console.
```
Comment (mbvictoria) [inline docs/troubleshooting-guide/bugs.md]: I think this should probably say "Metabase will send debugging information and errors to your browser's developer console."

File: docs/troubleshooting-guide/bugs.md [hunk 2]
```diff
@@ -1,47 +1,33 @@
...
+6. Which database is used
+7. What browser version
 
 ## Helpful tidbits
 
 ### Accessing the Metabase server logs
-While you can always look for the logs Metabase leaves on your server file system (or however you collect logs), if you are logged into Metabase with an admin account, you can also access them from the drop down menu in the upper right hand corner.
+While you can always look for the logs Metabase leaves on your server file system (or however you collect logs), if you are logged into Metabase with an admin account, you can also access them from the gear drop down menu in the upper right hand corner -> Admin -> Troubleshooting -> Logs.
 
-![Logs Screenshot](images/ServerLogs.png)
```
Comment (flamber) [inline docs/troubleshooting-guide/bugs.md]: I'm not sure if I should delete this file or if it would have consequences for earlier versions?

Comment (mazameli) [reply docs/troubleshooting-guide/bugs.md]: Should be safe to delete — we version the docs, so past versions should still have a copy of this image in their built doc directories (e.g. here https://metabase.com/docs/v0.28.0/troubleshooting-guide/images/ServerLogs.png).

File: docs/troubleshooting-guide/bugs.md [hunk 3]
```diff
@@ -1,47 +1,33 @@
...
+### Checking for Javascript console errors
+Metabase will send debugging information and errors to your browser's developer console.
```
Comment (mazameli) [inline docs/troubleshooting-guide/bugs.md]: I think we could abbreviate this section even more:

```
### Checking for Javascript console errors
Metabase will send debugging information and errors to your browser's developer console. To open the developer console, follow the instructions for your web browser of choice:

* [Chrome](https://developers.google.com/web/tools/chrome-devtools/open#console)
* [Firefox](https://developer.mozilla.org/en-US/docs/Tools/Web_Console/Opening_the_Web_Console)
* [Safari](https://support.apple.com/guide/safari-developer/develop-menu-dev39df999c1/mac)
* [Edge](https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium](https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium)

**Review by mazameli — APPROVED** [review]
Thanks, Flamber!

**Pull request merged**